js/utils/src/utils.svelte.ts

```diff
@@ -392,8 +392,7 @@ export class Gradio<T extends object = {}, U extends object = {}> {
 		this.last_update = tick();
 	}
 
-	async set_data(data: Partial<U & SharedProps>): Promise<void> {
-		await this.last_update;
```

Comment:
I think we need this because a bunch of unrelated e2e tests started failing. Let me see if I can add back